### PR TITLE
[IMP] point_of_sale: display current company partners

### PR DESCRIPTION
**Pull request by mgh-odoo**
**Task**: https://www.odoo.com/web#id=2145607&model=project.task&menu_id=4720
**Pad**: https://pad.odoo.com/p/r.7168a8e8210fa7cc810cdf56eb92dc19

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/point_of_sale/static/src/js/models.js [hunk 1]
```diff
@@ -202,6 +202,7 @@ exports.PosModel = Backbone.Model.extend({
         fields: ['name','street','city','state_id','country_id','vat','lang',
                  'phone','zip','mobile','email','barcode','write_date',
                  'property_account_position_id','property_product_pricelist'],
+        domain: function(self) { return [['company_id', 'in', [self.company && self.company.id, false]]] },
```
Comment (pimodoo) [inline addons/point_of_sale/static/src/js/models.js]: @mgh-odoo Could you modify the domain by ['company_id', '?=', self.company.id] and test that it works well.

Thanks

File: addons/point_of_sale/static/src/js/models.js [hunk 2]
```diff
@@ -620,7 +621,7 @@ exports.PosModel = Backbone.Model.extend({
     },
 
     prepare_new_partners_domain: function(){
-        return [['write_date','>', this.db.get_partner_write_date()]];
+        return [['company_id', 'in', [this.company && this.company.id, false]], ['write_date','>', this.db.get_partner_write_date()]];
```
Comment (pimodoo) [inline addons/point_of_sale/static/src/js/models.js]: same here

File: addons/point_of_sale/static/src/js/models.js (hunk 1 above)
Comment (mgh-odoo) [reply addons/point_of_sale/static/src/js/models.js]: @pimodoo it will not work in our case because we have some customers 'False' on the left side.

Comment (pimodoo) [reply addons/point_of_sale/static/src/js/models.js]: @mgh-odoo yes, the operator '=?' takes the value or False

Comment (mgh-odoo) [reply addons/point_of_sale/static/src/js/models.js]: @pimodoo reverted to the previous `['company_id', 'in', [self.company && self.company.id, false]` domain. Thanks!

**Comment by pimodoo** [comment]
@mgh-odoo Hello, not needed anymore since: https://github.com/odoo/odoo/commit/bd5305a51b9a264b971961aebbdc3e79590e4ad9

So I close this PR

**Pull request closed without merging**